docs/content/guide/animations.ngdoc

```diff
@@ -274,6 +274,98 @@ myModule.directive('my-directive', ['$animate', function($animate) {
 }]);
 ```
 
+## Animations on app bootstrap / page load
+
+By default, animations are disabled when the Angular app {@link guide/bootstrap bootstraps}. If you are using the {@link ngApp} directive,
+this happens in the `DOMContentLoaded` event, so immediately after the page has been loaded.
+Animations are disabled, so that UI and content are instantly visible. Otherwise, with many animations on
+the page, the loading process may become too visually overwhelming, and the performance may suffer.
+
+Internally, this is done by waiting for two digests - that is usually enough to wait until all templates
```

Comment:
I think we wait until all templates have been downloaded and **then** we wait for one digest more (plus the one that is already in progress).